### PR TITLE
Update AWS CIDR values

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2040

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ To create and use Workspace API tokens, see [Workspace API tokens](workspace-api
 ### Additional improvements
 
 - Port 80 is no longer used for certificate management on the data plane. 
+- The default CIDR range for new AWS clusters has been reduced from `/19` to `/20`.
```
Comment (jwitz) [inline astro/release-notes.md:37]:
```suggestion
```

File: astro/create-cluster.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ To create a new Astro cluster on Azure for your Organization, submit a request t
 - Your preferred node instance type.
 - Your preferred maximum node count.
 
-If you don't specify configuration preferences, Astronomer support creates a cluster with `Standard_D4d_v5 nodes`, one Postgres Flexible Server instance (`D4ds_v4`), and a maximum node count of 20 in `CentralUS`. If you're using Virtual Private Cloud (VPC) peering, a CIDR block (RFC 1918 IP Space) with the default CIDR range `172.20.0.0/19` is implemented.
+If you don't specify configuration preferences, Astronomer support creates a cluster with `Standard_D4d_v5 nodes`, one Postgres Flexible Server instance (`D4ds_v4`), and a maximum node count of 20 in `CentralUS`. If you're using Virtual Private Cloud (VPC) peering, a CIDR block (RFC 1918 IP Space) with the default CIDR range `172.20.0.0/20` is implemented.
```
Comment (chomipi88) [inline astro/create-cluster.md:152]: We are not changing this for Azure.

**Review by chomipi88 — COMMENTED** [review]
one fix. otherwise lgtm!

**Review by lzdanski — APPROVED** [review]
Style + grammar look good!

File: astro/create-cluster.md (hunk 2 above)
Comment (jwitz) [inline astro/create-cluster.md:152]:
```suggestion
If you don't specify configuration preferences, Astronomer support creates a cluster with `Standard_D4d_v5 nodes`, one Postgres Flexible Server instance (`D4ds_v4`), and a maximum node count of 20 in `CentralUS`. If you're using Virtual Private Cloud (VPC) peering, a CIDR block (RFC 1918 IP Space) with the default CIDR range `172.20.0.0/19` is implemented.
```

**Review by chomipi88 — APPROVED** [review]

**Pull request merged**